R/custom_gradient_color.R

```diff
@@ -15,14 +15,14 @@
 #'
 #' @examples
 #' custom_gradient_color(1, 0.5, 0.2)
-custom_gradient_color <- function(high, medium, low) {
+custom_gradient_color <- function(risk_score_high = 1, risk_score_medium = 1, risk_score_low = 1) {
   # define RGB values for "high," "medium," and "low"
-  high_color <- red <- c(1, 0, 0)
-  medium_color <- orange <- c(1, 0.5, 0)
-  low_color <- green <- c(0, 1, 0)
+  high_color <- red <- hex_to_rgb(high_hex())
+  medium_color <- orange <- hex_to_rgb(medium_hex())
+  low_color <- green <- hex_to_rgb(low_hex())
 
   # interpolate the colors based on proportions : 1 is highest intensity
-  final_color <- high_color * high + medium_color * medium + low_color * low
+  final_color <- high_color * risk_score_high + medium_color * risk_score_medium + low_color * risk_score_low
 
```

Comment:
Is the word "score" useful? Maybe `risk_high`, `risk_medium` and `risk_low` are clear enough?